doc/notebooks/example_notebook.ipynb

```diff
@@ -17,12 +17,115 @@
  "nbformat": 4,
  "nbformat_minor": 2,
  "cells": [
+  {
+   "source": [
+    "# Try GRASS GIS in Jupyter Notebook with Python\n",
+    "\n",
+    "[<img src=\"grass_gis_logo.png\" alt=\"GRASS GIS\" style=\"width:200px;\"/>](https://grass.osgeo.org/)\n",
+    "\n",
+    "This is a quick introduction into *GRASS GIS* in a *Jupyter Notebook* using the *Python* scripting language.\n",
+    "The interactive notebook is available online thanks to the [*Binder*](https://mybinder.org/) service.\n",
+    "\n",
+    "Examples here are using a sample GRASS GIS dataset for North Carolina, USA. The dataset is included in this environment.\n",
+    "\n",
+    "## Usage\n",
+    "\n",
+    "To run the selected part which is called a cell, hit `Shift + Enter`.\n",
+    "\n",
+    "## Start\n",
+    "\n",
+    "There is several ways how GRASS GIS can be used. When using Python in notebook, we usually find GRASS GIS Python packages, import them, initialize GRASS GIS session, and set several variables useful for using GRASS GIS in a notebook."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import Python standard library and IPython packages we need.\n",
+    "import os\n",
+    "import sys\n",
+    "import subprocess\n",
+    "from IPython.display import Image\n",
+    "\n",
+    "# Ask GRASS GIS where its Python packages are.\n",
+    "gisbase = subprocess.check_output([\"grass\", \"--config\", \"path\"]).strip()\n",
```

Comment:
You need to use the code from your GSoC evaluation period notebook. This does not work in Python 3.